keyboards/k34/readme.md

```diff
@@ -0,0 +1,19 @@
+# k34
+
+![k34](https://github.com/wongjingping/k34/raw/main/docs/k34.jpg?raw=true)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [wongjingping](https://github.com/wongjingping)
+All hardware supported can be found in the [k34 repository](https://github.com/wongjingping/k34)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Done